internal/log/stdLogger.go

```diff
@@ -0,0 +1,51 @@
+package log
+
+import (
+	"fmt"
+	"io"
+	goLog "log"
+	"os"
+	"runtime"
+)
+
+var stdLogger StdLogger
+
+type StdLogger struct {
+	Logger
+}
+
+func SetOutput(w io.Writer) {
+	goLog.SetOutput(w)
+}
+
+func Output(level string, format string, args ...interface{}) {
```

Comment:
This should be part of Standard and private:
```
func (_ Standard) output(…){…}
```
Or just private.

Reply:
I would make `Output` private; as we don't want anyone outside of the package directly call this function.